virttest/qemu_devices/qcontainer.py

```diff
@@ -1020,6 +1020,10 @@ def machine_s390_virtio(cmd=False):
             devices.append(qdevices.QStringDevice('machine', cmdline=cmd,
                                                   child_bus=bus,
                                                   aobject="virtio-blk-ccw"))
+            vga = params.get('vga')
+            if vga and vga != 'none':
+                # Always add mouse when 'vga' is set
+                devices.append(qdevices.QDevice("virtio-mouse-ccw"))
```

Comment:
The support of virtio input devices has been applied recently (https://github.com/avocado-framework/avocado-vt/pull/1737), so I think it is better to do this in that way.

Reply:
thank you, I wasn't aware of this.